src/features/session_trace/aggregate/index.js

```diff
@@ -189,13 +190,12 @@ export class Aggregate extends AggregateBase {
       retryDelay: this.harvestTimeSeconds
     }, this)
     this.#scheduler.harvest.on('resources', this.#prepareHarvest.bind(this))
-    if (dontStartHarvestYet === false) this.#scheduler.runHarvest({ needResponse: true }) // sends first stn harvest immediately
+    if (dontStartHarvestYet === false) this.#scheduler.runHarvest({}) // sends first stn harvest immediately
     startupBuffer.decide(true) // signal to ALLOW & process data in EE's buffer into internal nodes queued for next harvest
   }
 
   #onHarvestFinished (result) {
-    if (result.sent && result.responseText && !this.ptid) { // continue interval harvest only if ptid was returned by server on the first
-      this.agentRuntime.ptid = this.ptid = result.responseText
+    if (result.sent && !result.failed && !this.#scheduler.started) { // continue interval harvest only after first call
```

Comment:
Use `everSent` to keep sync with L220
```suggestion
    if (result.sent && !result.failed && !this.everSent) { // continue interval harvest only after first call
```